II.pass-tests/07-second-largest/second-largest.js

```diff
@@ -0,0 +1,6 @@
+function secondLargest(numbers) {
+    numbers.sort();
```

Comment:
You might need to be a little careful with the `.sort()` method here. By default it sorts **alphabetically** not **numerically**. That means if you're trying to sort an array of numbers it can do some weird things.

To see what I mean, try adding 230 to the `numbers` array in the test. Try looking up "js sort numbers" to get a fix

Reply:
Thanks. I fixed it.